### ISSUE TITLE
Fix string warning in t8_step1_coarsemesh.c

### ISSUE THREAD

**Issue by holke** [holke/t8code#100]
When compiling with pure C++ we get the following warning:

```bash
 example/tutorials/t8_step1_coarsemesh.c: In function ‘int main(int, char**)’:
example/tutorials/t8_step1_coarsemesh.c:96:32: warning: ISO C++ forbids converting a string constant to ‘char*’ [-Wwrite-strings]
   char               *prefix = "t8_step1_tetcube";
```

This should get resolved.

**Comment by holke** [comment]
Is this fixed now?

**Comment by lukasdreyer** [comment]
This was fixed in #124
